app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>This is detail of the user </h1>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap" rel="stylesheet"> 
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  </head>
+
+  <body>
+    <div class="root-container user-container">
+      <%= render 'shared/user_info', user: @user %>
+      <div class="bio-container">
+        <h3>Bio</h3>
+        <p class="bio"> <%=@user.bio%> </p>
+      </div>
+      <div class="posts">
+        <% @user.posts.each_with_index do |post, post_no| %>
+          <%= render 'shared/post_info', post: post, post_no: post_no %>
+        <%end%>
+        <button class="btn">See all posts</button>
+      </div>
```

Comment:
- Hi, can you make sure to use the #three_recent_posts method to render the user's 3 most recent posts? Because it is the requirement of this project.
- Please ensure that `See all post` link opens the user's post because I have struggled to open it manually. 
![2](https://github.com/Paul-tes/Blog-App/assets/63972670/dc2f68f5-9afe-4d2b-a666-829f1d588cac)
- Original
![4](https://github.com/Paul-tes/Blog-App/assets/63972670/81f3a98f-1d0c-42a4-932a-888a8c8e9456)